### ISSUE TITLE
Game Crashing if gdj files from Java sources are being used 

### ISSUE THREAD

**Issue by aless2003** [utopia-rise/godot-kotlin-jvm#578]
When you use gdj files which have been generated from Java Source Files, the game crashes on start with a NoClassDefFoundError, which is only shown when running the editor itself from the terminal.  The same gdj file, but generated from a Kotlin class works fine.

The project is basically just the standard project generated by the IntelliJ plugin with renamed packages.

Here's both the log and the project I've been using:
[log.txt](https://github.com/utopia-rise/godot-kotlin-jvm/files/14270778/log.txt)
[SuikaGame.zip](https://github.com/utopia-rise/godot-kotlin-jvm/files/14271152/SuikaGame.zip)

**Comment by aless2003** [comment]
Ok so turns out it's because of the source set. When using src/main/kotlin Java source files aren't built, if you use src/main/java however instead, both Kotlin and java are being compiled and the game works as expected

**Comment by chippmann** [comment]
Yeah that's normal java/kotlin behaviour and not related to this project. Still i guess we could change the starting project folders in the wizard to use a java folder.

**Comment by CedNaru** [comment]
Closing because of Scala support so a Java main src would make even less sense.
Each language can free use their own java/kotlin/scala sub directory and even set custom one in gradle and it will work just fine.
